### ISSUE TITLE
Refresh metadata when rescanning the library

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#3109]
Somewhere in the GUI there should be a higher profile option for refreshment of metadatas.

This is especially important since Mixxx organizes the tracks in the Library GUI according to ID3 data, but has no ID3 editor built in; there are too many steps to reordering the file presentation in the Library GUI after ID3 edits outside of Mixxx happen.

The right click option in Library is great and works, but it would be nice if there was a toggle option in preferences -> Library  to include metadata refresh as part of library rescan, or if metadata refresh was included in library rescan by default.

Or, it would be good if there was an option for metadata refresh/rescan in the library dropdown menu* (unless of course it becomes part of rescan by default in which case it would be unnecessary).

So basically, one of these:

1. "always reload metadata from files when refreshing library" becoming default

2. "reload metadata from files when refreshing library" added as an option in the library preferences menu

3. an entirely separate library dropdown menu entry specifically for refreshing all metadatas*

*If Bug #1085691 is addressed, then a Library dropdown entry could be replaced by a button in the analyze pane or similar.
